universal-login-react/src/App.tsx

```diff
@@ -141,13 +141,14 @@ export const App = () => {
                 }
                 const topUpProps: TopUpProps = {
                   contractAddress: Wallet.createRandom().address,
-                  onGasParametersChanged: console.log,
-                  sdk
+                  sdk,
```

Comment:
`topUpRegularProps` and `topUpDeploymentProps`

Reply:
Done.